### PR TITLE
simplify repl

### PR DESCRIPTION
**Pull request by benjyz**

### CODE REVIEW

**Review by jcburley — CHANGES_REQUESTED** [review]

File: repl.go [hunk 1]
```diff
@@ -125,11 +125,12 @@ func repl(phase Phase) {
 
 	reader := NewReader(runeReader, "<repl>")
 
+	username := GLOBAL_ENV.CurrentNamespace().Name.ToString(false)
```
[the remark this answers is not in the document]
Comment (benjyz) [reply repl.go]: done that, thanks for the feedback.

**Review by jcburley — APPROVED** [review]

**Comment by jcburley** [comment]
One other thing: `username` is not really a great name for this variable; perhaps `namespace` or similar?

**Comment by benjyz** [comment]
ok , changed the name

**Comment by candid82** [comment]
Thanks!

**Pull request merged**